### ISSUE TITLE
Build failed on Windows 10

### ISSUE THREAD

**Issue by mehrab2603** [mitsuba-renderer/mitsuba2#147]
I am trying to build the following variants:

- scalar_rgb
- gpu_autodiff_rgb

CUDA 10.2
Optix 7.0.0
VS 2019 16.5.2

I am getting the following errors:

```
[build] C:\Users\mehra\Documents\mitsuba2\src\films\hdrfilm.cpp(383,1): error C3646: 'm_mutex': unknown override specifier [C:\Users\mehra\Documents\mitsuba2\build\src\films\hdrfilm-obj.vcxproj]
[build] C:\Users\mehra\Documents\mitsuba2\src\films\hdrfilm.cpp(385): message : see reference to class template instantiation 'mitsuba::HDRFilm<Float,Spectrum>' being compiled [C:\Users\mehra\Documents\mitsuba2\build\src\films\hdrfilm-obj.vcxproj]
[build] C:\Users\mehra\Documents\mitsuba2\src\films\hdrfilm.cpp(383,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\mehra\Documents\mitsuba2\build\src\films\hdrfilm-obj.vcxproj]
[build] C:\Users\mehra\Documents\mitsuba2\src\films\hdrfilm.cpp(383,10): error C2039: 'mutex': is not a member of 'std' [C:\Users\mehra\Documents\mitsuba2\build\src\films\hdrfilm-obj.vcxproj]
```

Build successfully completes on Linux. Undoing commit 567ab3884c86dde5c043bad523b5aafa04cad537 results in successful build on Windows.

**Comment by Speierers** [comment]
Thanks for reporting this @mehrab2603 !

It's fixed now: 92c294e

**Comment by mehrab2603** [comment]
Hi @Speierers,
I am getting a new set of errors on the latest codebase.

```
[build] C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(231,5): warning MSB8065: Custom build for item "C:\Users\Rabbi\Documents\mitsuba2\build\CMakeFiles\654dfcecb315a617dc3b3cd414f1e540\rgb2spec_opt_run.rule" succeeded, but specified output "c:\users\rabbi\documents\mitsuba2\build\ext_build\rgb2spec\cmakefiles\rgb2spec_opt_run" has not been created. This may cause incremental build to work incorrectly. [C:\Users\Rabbi\Documents\mitsuba2\build\ext_build\rgb2spec\rgb2spec_opt_run.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(141,18): error C2065: 'uint': undeclared identifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(83): message : while compiling class template member function 'mitsuba::BlenderMesh<enoki::DiffArray<enoki::CUDAArray<float>>,mitsuba::Spectrum<enoki::DiffArray<enoki::CUDAArray<float>>,4>>::BlenderMesh(const mitsuba::Properties &)' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(306): message : see reference to class template instantiation 'mitsuba::BlenderMesh<enoki::DiffArray<enoki::CUDAArray<float>>,mitsuba::Spectrum<enoki::DiffArray<enoki::CUDAArray<float>>,4>>' being compiled [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(141,23): error C2146: syntax error: missing ';' before identifier 'p' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(141,24): error C2143: syntax error: missing ';' before '=' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(141,24): error C2143: syntax error: missing ')' before '=' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(141,1): error C2059: syntax error: '=' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(141,28): error C2065: 'p': undeclared identifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(141,15): error C2065: 'p': undeclared identifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(141,1): error C2059: syntax error: ')' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(142,77): error C2059: syntax error: ';' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(143,26): error C2065: 'p': undeclared identifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(146,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(146,21): error C2146: syntax error: missing ';' before identifier 'vertex_ctr' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(149,1): error C3646: 'normal': unknown override specifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(149,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(154,1): error C3646: 'uv': unknown override specifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(154,29): error C2059: syntax error: '{' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(154,1): error C2334: unexpected token(s) preceding '{'; skipping apparent function body [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(166,1): error C3646: 'value': unknown override specifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(166,30): error C2059: syntax error: '{' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(166,1): error C2334: unexpected token(s) preceding '{'; skipping apparent function body [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(172,1): error C3927: '->': trailing return type is not allowed after a non-function declarator [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(172,19): error C3484: syntax error: expected '->' before the return type [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(172,1): error C3613: missing return type after '->' ('int' assumed) [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(172,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(172,9): error C2371: 'mitsuba::vertex_map': redefinition; different basic types [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(170): message : see declaration of 'mitsuba::vertex_map' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(172,20): error C2146: syntax error: missing ';' before identifier 'resize' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,9): error C2059: syntax error: 'for' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,1): error C2143: syntax error: missing ')' before ';' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,2): error C2143: syntax error: missing ';' before '<' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,1): error C2143: syntax error: missing ';' before '++' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,20): error C2086: 'int mitsuba::tri_loop_id': redefinition [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175): message : see declaration of 'mitsuba::tri_loop_id' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,1): error C2059: syntax error: ')' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,1): error C2143: syntax error: missing ';' before '{' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(175,1): error C2447: '{': missing function header (old-style formal list?) [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(271,9): error C2059: syntax error: 'do' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(271,1): error C2143: syntax error: missing ';' before '{' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(271,1): error C2447: '{': missing function header (old-style formal list?) [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(271,1): error C2059: syntax error: 'while' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(272,9): error C2059: syntax error: 'if' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(275,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(275,27): error C2065: 'ScalarSize': undeclared identifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(275,39): error C2146: syntax error: missing ';' before identifier 'tmp_triangles' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(276,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(276,37): error C2065: 'UInt32': undeclared identifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(276,23): error C2923: 'mitsuba::DynamicBuffer': 'UInt32' is not a valid template type argument for parameter 'Value' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(276,1): error C2062: type 'unknown-type' unexpected [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(276,46): error C2039: 'copy': is not a member of '`global namespace'' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(278,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(278,26): error C2065: 'vertex_ctr': undeclared identifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(279,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(279,1): error C2653: 'FloatStorage': is not a class or namespace name [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(279,53): error C2065: 'tmp_vertices': undeclared identifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(279,48): error C3861: 'copy': identifier not found [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(280,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(280,1): error C2653: 'FloatStorage': is not a class or namespace name [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(280,51): error C2065: 'tmp_normals': undeclared identifier [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(280,46): error C3861: 'copy': identifier not found [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(281,9): error C2059: syntax error: 'if' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(282,1): error C2653: 'FloatStorage': is not a class or namespace name [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(283,9): error C2059: syntax error: 'if' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(283,1): error C2143: syntax error: missing ';' before '{' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(283,1): error C2447: '{': missing function header (old-style formal list?) [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(292,1): error C3927: '->': trailing return type is not allowed after a non-function declarator [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(292,20): error C3484: syntax error: expected '->' before the return type [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(292,1): error C3613: missing return type after '->' ('int' assumed) [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(292,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(292,9): error C2086: 'int mitsuba::m_faces_buf': redefinition [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(276): message : see declaration of 'mitsuba::m_faces_buf' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(292,21): error C2146: syntax error: missing ';' before identifier 'managed' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(293,1): error C3927: '->': trailing return type is not allowed after a non-function declarator [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(293,31): error C3484: syntax error: expected '->' before the return type [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(293,1): error C3613: missing return type after '->' ('int' assumed) [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(293,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(293,9): error C2086: 'int mitsuba::m_vertex_positions_buf': redefinition [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(279): message : see declaration of 'mitsuba::m_vertex_positions_buf' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(293,32): error C2146: syntax error: missing ';' before identifier 'managed' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(294,1): error C3927: '->': trailing return type is not allowed after a non-function declarator [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(294,29): error C3484: syntax error: expected '->' before the return type [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(294,1): error C3613: missing return type after '->' ('int' assumed) [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(294,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(294,9): error C2086: 'int mitsuba::m_vertex_normals_buf': redefinition [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(280): message : see declaration of 'mitsuba::m_vertex_normals_buf' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(294,30): error C2146: syntax error: missing ';' before identifier 'managed' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(295,9): error C2059: syntax error: 'if' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(298,1): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(299,9): error C2059: syntax error: '}' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
[build] C:\Users\Rabbi\Documents\mitsuba2\src\shapes\blender.cpp(299,1): error C2143: syntax error: missing ';' before '}' [C:\Users\Rabbi\Documents\mitsuba2\build\src\shapes\blender-obj.vcxproj]
```

**Comment by Speierers** [comment]
My bad, let me fix this.

**Comment by Speierers** [comment]
Should be fixed now: (2394ee98)
